smoke-tests/spec/alertmanager_spec.rb

```diff
@@ -1,12 +0,0 @@
-require "spec_helper"
```

Comment:
Do we have an identical alertmanager check in go?

Reply:
This particular check confirms if the correct servicemonitor exists: We do it slightly differently in Go, checking if services and servicemonitors exist in bulk
https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/main/test/config/config.go#L102-L111